### PR TITLE
chore(C): added bit-manipulation

### PR DESCRIPTION
**Pull request by ven0m-hue**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

Added three files in the directory bit-manipulation 
- Add and Subtract two integers without using a '+' operator. 
- Multiply two integers without using a '*' operator. 
- Divide two integers without using a '/' operator.

### CODE REVIEW

**Review by aayushjain7 — APPROVED** [review]
looks good to me 👍🏼

File: algorithms/C/bit-manipulation/multiply-bitwise.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Shift and add algo for multiplication wihtout using multiplication operator
```
Comment (aayushjain7) [inline algorithms/C/bit-manipulation/multiply-bitwise.c:1]: change `wihtout` to `without`

**Review by aayushjain7 — CHANGES_REQUESTED** [review]

File: algorithms/C/bit-manipulation/multiply-bitwise.c (hunk 1 above)
Comment (ven0m-hue) [reply algorithms/C/bit-manipulation/multiply-bitwise.c]: Done. Thanks.

**Review by aayushjain7 — APPROVED** [review]
good work!

**Pull request merged**